templates/layout.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}Sign Language Translator{% endblock %}</title>
+    <style>
```

Comment:
Preferred: Make a separate css file (preferrably in a different folder) to holder your styling changes.